main.py

```diff
@@ -115,6 +121,15 @@ def handle_pending_requests():
             del pending_requests[key_to_delete]
 
 
+def create_usage_dict(value, choices):
+    usage_dict = {
+        "prompt_tokens": len(encoding.encode(value["prompt"])),
+        "completion_tokens": sum([len(encoding.encode(choice["text"])) for choice in choices])
```

Comment:
The OpenAI API will set `completion_tokens` to `null` if all completions were zero in length. Does langchain rely on this behavior?